zuul.d/jobs.yaml

```diff
@@ -1065,3 +1065,10 @@
       inventory_name: orange
     secrets:
       - orange_credentials
+
+- job:
+    name: spark-funcional-test-k8s
```

Comment:
The job download Java binary Spark release directly, so please use `spark-bin-*` as job name, currently all of other OpenLab jobs compile project from source code